react-app/src/app/constants.js

```diff
@@ -1,18 +1,45 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { faHome, faDiceOne, faUser, faSignOutAlt, faQuestion } from '@fortawesome/free-solid-svg-icons';
 
+import { removeState } from '../redux/localStorage';
+import { logout } from '../redux/Auth/actions';
+
+// eslint-disable-next-line
+const handleLogout = () => {
+  removeState('token');
+  removeState('tokenExpireDateTime');
+  // TODO: why does not work?
+  logout();
+};
+
 library.add(faHome, faDiceOne, faUser, faSignOutAlt, faQuestion);
 
-export const LINKS = [
-  { name: 'Home', to: '/', icon: 'home', restrict: false },
-  { name: 'Tic-Tac-Toe', to: '/app', icon: 'dice-one', restrict: true },
-  { name: 'Dashboard', to: '/user', icon: 'user', restrict: true },
-  { name: 'Logout', to: '/', icon: 'sign-out-alt', restrict: true, isLogout: true },
-  { name: 'Help', to: '/help', icon: 'question', restrict: false }
-];
+export const ROUTES = [];
+ROUTES.home = { name: 'Home', route: '/' };
+ROUTES.app = { name: 'Game', route: '/app' };
+ROUTES.login = { name: 'Login', route: '/login' };
+ROUTES.help = { name: 'Help', route: '/help' };
+ROUTES.user = { name: 'Dashboard', route: '/user' };
+ROUTES.points = { name: 'Points', route: '/user/points' };
+ROUTES.history = { name: 'History', route: '/user/history' };
+
+export const LINKS = {
+  topBar: [
+    { name: ROUTES.home.name, to: ROUTES.home.route, icon: 'home', restrict: false },
+    { name: ROUTES.app.name, to: ROUTES.app.route, icon: 'dice-one', restrict: true },
+    // { name: ROUTES.user.name, to: ROUTES.user.route, icon: 'user', restrict: true },
+    { name: 'Logout', to: ROUTES.login.route, icon: 'sign-out-alt', restrict: true, onClick: handleLogout },
```

Comment:
In my opinion it shouldnt have a `to` attribute, since logging out is enough for redirecting to the app home